average/average.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8" />
+</head>
+
+<body style="text-align: center;">
+    <h1 style="color:green;">Student's cost of education</h1>
+    <h3>Input name and scores of a student</h3>
+    <form name="name">
+        <p><input type="text" id="name" placeholder="enter name"></input></p>
+    </form>
+    <form name="score" id="score">
+        <p><input type="text" id="score1" value="" placeholder="enter  score"></input></p>
+        <p><input type="text" id="score2" value="" placeholder="enter  score"></input></p>
+        <p><input type="text" id="score3" value="" placeholder="enter  score"></input></p>
+        <p><input type="text" id="score4" value="" placeholder="enter  score"></input></p>
+        <p><input type="text" id="score5" value="" placeholder="enter  score"></input></p>
+    </form>
+    <form>
+        <p><input onclick="inC()" type="button" id="but" value="add" /></p>
+    </form>
+    <div id="printBlock"></div>
+    <script type="text/javascript">
```

Comment:
rename file to index.html. move JS script to separate file and include it to index.